### DISCUSSION TITLE
关于io_uring

### DISCUSSION THREAD

**Discussion by samoyedsun** [cloudwu/skynet#1661] [Ideas]
很多人都说io_uring比epoll效率高，云大有没有计划加一个io_uring的网络模块，发现asio就加了一个io_uring代替epoll的选项

**Comment by cloudwu** [comment]
如果有明确的数据表示 io 成为了性能瓶颈才考虑。

否则我认为没有意义。

**Comment by cloudfreexiao** [comment]
io_uring 都是作为可选项 毕竟对内核版本有要求
